HeliPort/Appearance/StatusMenu.swift

```diff
@@ -552,6 +563,13 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
     @objc private func updateNetworkList() {
         guard isNetworkCardEnabled else { return }
 
+        networkItemList.forEach {
+            guard let wifiView = $0.view as? WifiMenuItemView, !wifiView.isMouseOver else {
+                Log.debug("Skipping network list since mouse over network")
+                return
+            }
+        }
+
```

Comment:
I personally think instead of blocking the refresh, appending the newly discovered SSIDs at the bottom of the list would provide a better user experience. However, that would require a lot more changes. Maybe it's better to merge this PR first, then enhance it in a separate PR?

Reply:
Yup I can remove this. We can figure out in another pull request.